app/views/shared/_common_nav_bar_menus.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render 'shared/my_school_menu' if can?(:read, :school_menu) %>
+<%= render 'shared/my_school_menu' if can?(:read, :school_menu) && current_user.school.active? %>
```

Comment:
Trying to think whether we can have this just as an ability rather than checking active school here

Reply:
`can?(:my_school_menu, current_user.school?)`

Reply:
Hmmm. Yes. Another instance of the cancan approach not quite fitting with our multi-tennant app.